srcpkgs/itch/template

```diff
@@ -0,0 +1,61 @@
+# Template file for 'itch'
+pkgname=itch
+version=1.18.0
+revision=1
+archs="i686 x86_64"
+build_style=go
+hostmakedepends="wget"
+makedepends="curl"
+short_desc="Itch.io indie games client for Linux"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="MIT"
+homepage="https://github.com/itchio/itch"
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64)
+		_dist_arch="amd64"
+		_filename=itch-setup
+		checksum="
+ e86c70044c7d02754c6e52fb428d1278678d729c77b8359b5d82198595da1351
+ 874a17d72a3bce1dda9b43ea6450a2f4454bf2b01207ae259a981b0f915bbedb"
+
+		;;
+	i686)
+		_dist_arch="386"
+		_filename=itch-setup
+		checksum="
+ e86c70044c7d02754c6e52fb428d1278678d729c77b8359b5d82198595da1351
+ 447940ea9cd7051e43f98b5dd5fdb0de94e30e2e479d2812314820030f876db2"
+
+;;
+esac
+
+_itchUrl="https://broth.itch.ovh/itch-setup/linux-${_dist_arch}/${version}/unpacked/default"
+
+distfiles="
```

Comment:
Move distfiles and wrksrc up before your Arch specific case statement

Reply:
I will move to the right position.

Reply:
The code was fixed and changed.